likecoin/js/admin-settings/src/store/site-matters-store.js

```diff
@@ -46,12 +46,20 @@ const actions = {
       return error;
     }
   },
+  * siteMattersLogout() {
+    try {
+      const response = yield { type: 'MATTERS_LOGOUT' };
+      return response;
+    } catch (error) {
+      console.error(error);
+      return error;
+    }
+  },
   * postSiteMattersOptions(options) {
-    yield { type: 'POST_MATTERS_LOGIN', data: options };
+    yield { type: 'POST_SITE_MATTERS_OPTIONS_TO_DB', data: options };
```

Comment:
I accidently delete this at last commit. Hence can't save publish options. Fix in the current commit.